modules/creation/tutorial.md

```diff
@@ -89,13 +89,13 @@ class MyModule extends Module
 
         parent::__construct();
 
-        $this->displayName = $this->l('My module');
-        $this->description = $this->l('Description of my module.');
+        $this->displayName = $this->trans('My module', [], 'Modules.MyModule.Admin');
```

Comment:
My understanding is that the domain should be:

`Modules.Mymodule.Admin`. 

I remember some issues with using camelCase inside the domains. cc @Oksydan do you remember something?

Reply:
Yes, if module name is `my_module_name`, the domain should be `Modules.Mymodulename.Admin`

Reply:
@Kaikina :) let's change `MyModule` to `Mymodule`

Reply:
Yep, camelCase domain names don't work properly. 
It isn't working because `DomainHelper::buildModuleBaseDomain` is responsible for formatting. It's removing `ps_` form module name, removing `_` characters and returning this value with `ucfirst`.  
So for module with name `mymodule` it's expecting `Mymodule`.